### ISSUE TITLE
Airway Breaks (1.3.1)

### ISSUE THREAD

**Issue by KSanders7070** [drchuck59/SCTBuilder#33]
Starting this issue again as a new issue to start fresh.

Example: V200 starts at ENI and ends at FMG. It then picks back up at BVL and ends at RLG.
Your parser draws a line between FMG and BVL still.

In the AWY.txt file, a break is indicated by an "X" on column 107 in the AWY1 lines:
```
L AN 0001 00107 AR11 AIRWAY GAP FLAG INDICATOR
'X' ENTERED WHEN AIRWAY DISCONTINUED
```
![V200 Not Breaking](https://user-images.githubusercontent.com/63331106/86632778-d30f7e00-bf9d-11ea-988c-920290fff701.JPG)

**Comment by drchuck59** [comment]
Defect corrected in 1.3.3 - close issue if resolved.

**Comment by KSanders7070** [comment]
This appears to be fixed now. Great job!
